plugins/gradle/README.md

```diff
@@ -11,9 +11,12 @@ This document describes the gradle plugin for graphql-java-codegen.
 This Gradle plugin is able to generate the following classes based on your GraphQL schema:
 * Interfaces for GraphQL queries, mutations and subscriptions
 * Interfaces for GraphQL unions
-* POJO classes for GraphQL types
-* Enum classes for each GraphQL enum
-* Interface Resolvers for GraphQL type fields 
+* POJO classes for GraphQL types/inputs
+* Enum classes for GraphQL enums
+* Interface Resolvers for GraphQL type fields
```

Comment:
I don't quite get this one, is it the same as the last?

Reply:
You are totally right.
I removed the last statement.